rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java

```diff
@@ -140,7 +142,7 @@ private Boolean isFileExisting(String pathname) {
     }
 
     private Boolean isFilePathValid(String pathname) {
-        String regex = "^(\\/)?([^\\/\\\u0000]+(\\/)?)+\\.xml";
+        String regex = "^(?>[a-z]:)?(?>\\\\|\\/)?([^\\\\\\/?%*:|\\\"<>\\r\\n]+(?>\\\\|\\/)?)+$";
```

Comment:
Can you rename this variable into something more expressive like correctPathnameRegex

Reply:
@tobwiens I addressed your comment. Thanks a lot!